tests/CMakeLists.txt

```diff
@@ -68,48 +71,62 @@ function(ConfigureTestInternal TEST_NAME)
       PROPERTY ADDITIONAL_CLEAN_FILES ${COVERAGE_CLEAN_FILES})
   endif()
 
-  add_test(NAME ${TEST_NAME} COMMAND ${TEST_NAME})
-
-  install(
-    TARGETS ${TEST_NAME}
-    COMPONENT testing
-    DESTINATION bin/gtests/librmm
-    EXCLUDE_FROM_ALL)
 endfunction()
 
 # Wrapper around `ConfigureTestInternal` that builds tests both with and without per thread default
 # stream
 function(ConfigureTest TEST_NAME)
+
+  set(options)
+  set(one_value GPUS PERCENT)
+  set(multi_value)
+  cmake_parse_arguments(_RMM_TEST "${options}" "${one_value}" "${multi_value}" ${ARGN})
+  if(NOT DEFINED _RMM_TEST_PERCENT)
```

Comment:
Is this is implicitly assuming all tests are SG? Do we want that?

Reply:
Unless explicitly specified with `GPUS 2` we presume all tests are single gpu.

Reply:
Updated. If only `GPUS` is provided we presume `100%`. If only percent is provided we presume a single gpu.